### PR TITLE
New client packages (also tar now) that don't use the config repo

### LINKED ISSUE #68: Provide tarball with client configuration

### PR DESCRIPTION
**Pull request by bedroge**
Closes #68 and partially #59.

See https://github.com/bedroge/filesystem-layer/releases/tag/v0.3.0 for an example of a release that was generated using the code of this branch.

### CODE REVIEW

**Comment by bedroge** [comment]
Also closes #69 , since this package won't depend on the cvmfs-config repo.

**Comment by bedroge** [comment]
I asked for a bit of advice on the CVMFS forum:
https://cernvm-forum.cern.ch/t/best-practices-for-client-configuration-packages-without-config-repo/47

As discussed there, I'm setting `CVMFS_HTTP_PROXY=DIRECT` in case the client hasn't set this parameter yet, but does have `CVMFS_CLIENT_PROFILE=single`.

Note that I'm still including multiple public keys here, since not all repos share the same masterkey yet and this still requires a change in the Ansible role (for which I opened a PR).

File: README.md [hunk 1]
```diff
@@ -166,9 +166,13 @@ Next, you need to make a file `/etc/cvmfs/default.local` manually; this file is
 contains, for instance, the URL to your local proxy and the size of the local cache. As an example, you can put
 the following in this file, which corresponds to not using a proxy and setting the local quota limit to 40000MB:
 ```
-CVMFS_HTTP_PROXY=DIRECT
+CVMFS_CLIENT_PROFILE=single
```
Comment (boegel) [inline README.md:169]: @bedroge Can you open an issue to remind ourselves to properly document why we do this, with pointers to the CVMFS docs?

Comment (bedroge) [reply README.md:169]: Done! #77

File: .github/workflows/test-playbooks.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@ jobs:
...
       - name: Execute the playbook
         run: docker exec ${{ matrix.component }}-${{ matrix.os }} /test-playbook.sh ${{ matrix.component }}.yml
-      - name: Execute additional playbook for Stratum 0
-        run: docker exec ${{ matrix.component }}-${{ matrix.os }} /test-playbook.sh ${{ matrix.component }}-deploy-cvmfs-config.yml
-        if: ${{ matrix.component == 'stratum0' }}
+#      - name: Execute additional playbook for Stratum 0
+#        run: docker exec ${{ matrix.component }}-${{ matrix.os }} /test-playbook.sh ${{ matrix.component }}-deploy-cvmfs-config.yml
+#        if: ${{ matrix.component == 'stratum0' }}
```
Comment (boegel) [inline .github/workflows/test-playbooks.yml:39]: @bedroge Why was this commented out?

File: README.md (hunk 1 above)
Comment (boegel) [reply README.md:169]: see #77

File: .github/workflows/test-playbooks.yml (hunk 2 above)
Comment (bedroge) [reply .github/workflows/test-playbooks.yml:39]: I've removed that playbook, so this playbook/test is not needed anymore. I'll remove these lines.

Comment (bedroge) [reply .github/workflows/test-playbooks.yml:39]: This was the playbook that generated the configuration files for our config repo, so we don't need it anymore.

File: inventory/group_vars/all.yml [hunk 3]
```diff
@@ -71,8 +72,19 @@ eessi_cvmfs_repositories:
       - CVMFS_AUTO_GC=false
       - CVMFS_AUTO_TAG_TIMESPAN="30 days ago"
       - CVMFS_GARBAGE_COLLECTION=true
-    client_options:
-      - CVMFS_NFILES=4096
+    client_options: []
+#      - CVMFS_NFILES=4096
```
Comment (boegel) [inline inventory/group_vars/all.yml:76]: Why was this commented out?

Comment (bedroge) [reply inventory/group_vars/all.yml:76]: Good question! :-)
To be honest, I wasn't even sure why it was there in the first place (it's a client-specific setting), and I don't see it in any other config packages, so I've commented these lines. I'll just remove them.

Comment (boegel) [reply inventory/group_vars/all.yml:76]: Probably just copied from https://github.com/galaxyproject/ansible-cvmfs/blob/master/README.md#example-playbook...

File: .github/workflows/test-playbooks.yml (hunk 2 above)
Comment (boegel) [reply .github/workflows/test-playbooks.yml]: Right, makes sense.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**